packages/terra-collapse/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "terra-collapse",
+  "main": "lib/Collapse.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Collapse component that transitions content in and out with the click on a button.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-collapse",
+    "Collapse",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
```

Comment:
Updated in this package: https://github.com/cerner/terra-core/pull/475/commits/199038e9d93dc96e64c428906beb0fcb63d97336

Logged https://github.com/cerner/terra-core/issues/480 to review other packages.